deploy/requirements.yml

```diff
@@ -10,7 +10,7 @@
 
 - src: https://github.com/caktus/ansible-role-k8s-web-cluster
   name: caktus.k8s-web-cluster
-  version: v1.3.0
+  version: v1.4.0
 
 - src: https://github.com/caktus/ansible-role-k8s-hosting-services
   name: caktus.k8s-hosting-services
```

Comment:
Maybe update caktus.k8s-hosting-services to v0.9.0 and re-run the deploy-hosting-services.yaml playbook? It includes helpful defaults for New Relic

Reply:
Done and done.